organizers/2018-06-25.html

```diff
@@ -0,0 +1,140 @@
+---
+layout: organizers
+---
+<!-- Syntax reference: https://github.com/hakimel/reveal.js/blob/master/index.html -->
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>June 25th 2018</h2>
+    <img data-src="/img/django-logo-negative.png" style="width:40%" alt="Django logo" class="plain"></img>
+</section>
+
+<section>
+    <h1>Hello, I'm Marco!</h1>
+    <h2>👋😆</h2>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Festicket</strong></li>
+        <li><strong>Thread</strong></li>
+        <li><strong>Time Out</strong></li>
+    </ul>
+</section>
+
+<section data-background='#eee'>
+    <img data-src="/img/festicket.svg" style="width:40%" alt="Festicket" class="plain"></img>
```

Comment:
use the new festicket logo https://www.festicket.com/public/logo5309b0327a478452f37be927bf15263c.png